.github/workflows/e2e.yml

```diff
@@ -15,8 +15,10 @@ permissions:
 
 jobs:
   cypress:
-    name: Cypress with Percy
+    name: Cypress with Happo
     runs-on: ubuntu-latest
+    env:
+      HAPPO_NONCE: '${{ github.event_name }}-${{ github.sha }}'
```

Comment:
While confusing, I think this is what's causing the 500 errors. The max length of this string in the database is 50, and I measure this at 53 chars. The fact that we're not validating this on the server is sloppy, I'll make sure we fix that. In the meantime, I think you can change this to 

```yaml
HAPPO_NONCE: '${{ github.run_id }}-${{ github.run_attempt }}'
```

Reply:
I've update the API on happo.io to send a 400 error with a useful error message in case the nonce is too long.